docs/scratch_orgs.rst

```diff
@@ -1,38 +1,253 @@
+
+To remove an existing default:
 
-Import an org from the Salesforce CLI
+.. code-block:: console
+
+    $ cci org default dev --unset
+
+
+
+Configure Predefined Orgs
+-------------------------
+
+Projects can customize the set of configurations available out of the box, and add further predefined orgs to meet project-specific needs. 
+
+An org configuration has a name, such as ``dev`` or ``qa``, and is defined by options set in ``cumulusci.yml`` as well as the contents of a specific ``.json`` scratch org definition file in the ``orgs`` directory. For orgs like ``dev`` and ``qa`` that are predefined for all projects, the configuration is located in the CumulusCI standard library, but can be customized by projects in ``cumulusci.yml``.
+
+Many projects that build managed packages offer a ``dev_namespaced`` org, a developer org that has a namespace. This org is defined under the  ``orgs`` section in ``cumulusci.yml``.
+
+.. code-block:: yaml
+
+    orgs:
+        scratch:
+            dev_namespaced:
+                config_file: orgs/dev.json
+                days: 7
+                namespaced: True
+
+This org uses the same Salesforce DX configuration file as the ``dev`` org, but has different configuration in ``cumulusci.yml``, resulting in a different org shape and a different use case. The key facets of the org shape that are defined in ``cumulusci.yml`` are whether or not the org has a namespace and the length of the org's lifespan. 
+
+Org definition files stored in the ``orgs`` directory are configured as specified in the `Salesforce DX Developer Guide <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file.htm>`_.
+
+Many projects never add a new org definition ``.json`` file, and instead add specific features and settings to the files shipped with CumulusCI. However, new definitions can be added and referenced under the ``orgs__scratch`` section of the ``cumulusci.yml`` file to establish org configurations that are completely customized for a project.
+
+
+
+Import an Org from the Salesforce CLI
 -------------------------------------
 
-Use a non-default Dev Hub
+CumulusCI can import existing orgs from the Salesforce CLI keychain.
+
+.. code-block:: console
+
+    $ cci org import sfdx_alias cci_alias
+
+For ``sfdx_alias``, specify the alias or username of the org in the Salesforce CLI keychain. For ``cci_alias``, provide the name to use in CumulusCI's keychain.
+
+.. note :: CumulusCI cannot automatically refresh orgs imported from Salesforce CLI when they expire.
+
+
+
+Use a Non-Default DevHub
 -------------------------
+
+By default, CumulusCI creates scratch orgs using the DevHub org configured as the ``defaultdevhubusername`` in ``sfdx``. Switch to a different DevHub org within a project by configuring the ``devhub`` service.
+
+.. code-block:: console
+
+    $ cci service connect devhub --project
+    Username: <DevHub username>
+    devhub is now configured for this project.
```

Comment:
Can someone please confirm that there is an actual period in this last line of code-block text.

Reply:
Confirmed.